pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -84,7 +85,39 @@ public class KitUtil {
                     + "LEFT JOIN carrier_service cs_to on (dkc.carrier_service_to_id=cs_to.carrier_service_id) "
                     + "LEFT JOIN carrier_service cs_return on (dkc.carrier_service_return_id=cs_return.carrier_service_id) "
                     + "LEFT JOIN kit_return_information ret on (dkc.kit_return_id=ret.kit_return_id) "
-                    + "LEFT JOIN kit_type t on (request.kit_type_id = t.kit_type_id) where ex.ddp_participant_exit_id is null";
+                    + "LEFT JOIN kit_type t on (request.kit_type_id = t.kit_type_id) where ex.ddp_participant_exit_id is null "
+                    + "and ddp_kit_request_settings_id is not null";
```

Comment:
only return kits that we can get their kit settings directly. This will make sure there is no overlap

Reply:
You mean no overlap with subkits?  If so, this is a generally useful remark so please enshrine it in the code so that those who come after us will always know why it's here :)